src/standalone/Soil/energy_hydrology.jl

```diff
@@ -63,22 +57,23 @@ struct EnergyHydrologyParameters{
 end
 
 function EnergyHydrologyParameters{FT}(;
-    ν::FT,
-    ν_ss_om::FT,
-    ν_ss_quartz::FT,
-    ν_ss_gravel::FT,
+    ν::F,
+    ν_ss_om::F,
+    ν_ss_quartz::F,
+    ν_ss_gravel::F,
     hydrology_cm::C,
-    K_sat::FT,
-    S_s::FT,
-    θ_r::FT,
+    K_sat::F,
+    S_s::F,
+    θ_r::F,
     PAR_albedo = FT(0.2),
```

Comment:
These can vary spatially, but the default is a scalar (not needed in configurations). When it is a scalar, we broadcast.